src/command.c

```diff
@@ -723,7 +735,7 @@ static int dv_convert_fc_v0(struct device *dev, struct dmp_cmb *cmb,
 }
 
 int dv_convert_fc_command(struct device *dev, struct dmp_cmb *cmb,
-			  struct dmp_dv_kcmd *cmd_info)
+			struct dmp_dv_kcmd *cmd_info)
```

Comment:
misalignment, either break line just after the bracket or align next line just after the bracket